examples/ci_multihead.json

```diff
@@ -21,7 +21,7 @@
             "radius": 2.0,
             "max_neighbours": 3,
             "hidden_dim": 4,
-            "num_conv_layers": 2,
+            "num_conv_layers": 3,
```

Comment:
Is this to improve accuracy?

Reply:
Yeah, tests passed with 3 conv layers on my computer. Need to update these parameters again though. The CI tests failed.